Visual Studio Project Template C#/PluginInfrastructure/GatewayDomain.cs

```diff
@@ -63,14 +63,19 @@ public int Value
     /// </summary>
     public class Position : IEquatable<Position>
     {
-        private readonly int pos;
+        private readonly long pos;
 
-        public Position(int pos)
+        public Position(long pos)
```

Comment:
position types need to be atleast 64-bit wide which can accommodate both 32-bit data in x86 execution and 64-bit data in x64 execution.
Hence long is used instead of int.